### PR TITLE
Corrected Chapter Titles and Links

### PR DESCRIPTION
**Pull request by cpholguera**
This PR covers:
- corrected "* Basic Security Testing" chapter titles in README and SUMMARY
- corrected 0x06e chapter title
- corrected title case in 0x05g
- corrected 2 web links (including parenthesis) that were properly rendered by GitHub but not by GitBook
- corrected chapter title in references and added link. It's not called "Testing Authentication and Session Management" but "Mobile App Authentication Architectures"
- reviewed the current 73 .md links and fixed the broken ones, such as:
  Screenshot 2019-08-10 at 09 44 09
- added more .md links for better interactivity

### CODE REVIEW

File: Document/0x06e-Testing-Cryptography.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## iOS Cryptography APIs
+## iOS Cryptographic APIs
```
Comment (cpholguera) [inline Document/0x06e-Testing-Cryptography.md]: Now it matches the Android chapter and the README/SUMMARY

**Comment by OMTGreleaser** [comment]
Results for commit dd134234c88500b96af0bbb4771d2ff7a19dc881: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 5f1b3adb1ca18a9d3c2b989695cd31f113d4581c: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit b126ec804eaa780f2b84fb0558057821b2256da6: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 537e1baa89f1a41011159915fc529a203f8f2f03: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 088078dbf54e01f77fb3c3aff7a0293bbd22e7af: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 690cdfd7a6205858414f35fe0441a54f172512d4: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x05g-Testing-Network-Communication.md [hunk 2]
```diff
@@ -357,7 +357,7 @@ This command will search for all methods that take a string and a variable list
 
 Hook each method with Frida and print the arguments. One of them will print out a domain name and a certificate hash, after which you can modify the arguments to circumvent the implemented pinning.
 
-### Testing the Network Security Configuration settings (MSTG-NETWORK-4)
+### Testing the Network Security Configuration Settings (MSTG-NETWORK-4)
```
Comment (commjoen) [inline Document/0x05g-Testing-Network-Communication.md]: Please change the compliance docs as well as all the links are fixed recently

File: Document/0x05h-Testing-Platform-Interaction.md [hunk 3]
```diff
@@ -39,14 +39,14 @@ The [following changes](https://developer.android.com/about/versions/pie/android
 
 - **Restricted access to call logs**: `READ_CALL_LOG`, `WRITE_CALL_LOG`, and `PROCESS_OUTGOING_CALLS` (dangerous) permissions are moved from `PHONE` to the new `CALL_LOG` permission group. This means that being able to make phone calls (e.g. by having the permissions of the `PHONE` group granted) is not sufficient to get access to the call logs.
 - **Restricted access to phone numbers**: apps wanting to read the phone number require the `READ_CALL_LOG` permission when running on Android 9 (API level 28).
-- **Restricted access to Wi-Fi location and connection information**: SSID and BSSID values cannot be retrieved (e.g. via [`WifiManager.getConnectionInfo`](https://developer.android.com/reference/android/net/wifi/WifiManager#getConnectionInfo() "WifiManager.getConnectionInfo") unless *all* of the following is true:
+- **Restricted access to Wi-Fi location and connection information**: SSID and BSSID values cannot be retrieved (e.g. via [`WifiManager.getConnectionInfo`](https://developer.android.com/reference/android/net/wifi/WifiManager#getConnectionInfo%28%29 "WifiManager.getConnectionInfo") unless *all* of the following is true:
```
Comment (commjoen) [inline Document/0x05h-Testing-Platform-Interaction.md]: Which url is working?

File: Document/0x06e-Testing-Cryptography.md (hunk 1 above)
Comment (commjoen) [reply Document/0x06e-Testing-Cryptography.md]: Can you please check if it is referenced from the compliance checklist?

File: Document/0x05h-Testing-Platform-Interaction.md (hunk 3 above)
Comment (cpholguera) [reply Document/0x05h-Testing-Platform-Interaction.md]: both URLs (including parenthesis) work and were properly rendered by GitHub but not by GitBook. So I replaced the parenthesis by `%28%29` and now they're properly rendered and also work.

File: Document/0x05g-Testing-Network-Communication.md (hunk 2 above)
Comment (cpholguera) [reply Document/0x05g-Testing-Network-Communication.md]: done

**Comment by OMTGreleaser** [comment]
Results for commit 2880eebb987951cf236a85dd68cdad7ae24a7c3a: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit dd08e8e4b643f428cd88b09cda51af231155b9e2: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 3c58dc57a19c29b2c01ea6a3d57999ab11797d0b: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**